src/backend/vulkan/src/lib.rs

```diff
@@ -650,11 +652,65 @@ impl queue::QueueFamily for QueueFamily {
     }
 }
 
+pub struct PhysicalDeviceMultiviewProperties {
+    pub max_multiview_view_count: u32,
+    pub max_multiview_instance_index: u32,
+}
```

Comment:
Substitute to `vk::PhysicalDeviceMultiviewProperties` without any pointers that would prevent `PhysicalDeviceProperties` below from being `Sync`.